vividus-plugin-rest-api/src/main/java/org/vividus/bdd/transformer/JsonRestApiTableTransformer.java

```diff
@@ -54,8 +55,9 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
                 entry("url", this::getJsonByUrl),
                 entry("variable", bddVariableContext::getVariable));
 
-
-        Map<String, String> columnsPerJsonPaths = Splitter.on(';').withKeyValueSeparator('=').split(columns);
+        Map<String, String> columnsPerJsonPaths = Stream.of(StringUtils.split(columns, ';'))
```

Comment:
Just curious in the ticket mentioned this way `withKeyValueSeparator(Splitter.on("=").limit(2))`

Reply:
@valfirst ^^